docs/topics/minishift-install-prereq.adoc

```diff
@@ -13,6 +13,19 @@ $ minishift version
 Minishift version: 1.0.0
 ----
 
+. Determine the command to install the correct version of the `oc` binary and run the command. 
```

Comment:
This doesn't really _install_ `oc`, it just adds correct path to $PATH. I'd suggest rewording to something like "Use `minishift oc-env` to add the correct `oc` binary to your path."

(The following item already uses the expression "your path", so I figured it should be enough. If a link to explanation is needed, this Wikipedia page is good: https://en.wikipedia.org/wiki/PATH_(variable) :-) )